management/server/src/main/java/org/apache/karaf/management/internal/Activator.java

```diff
@@ -60,6 +60,8 @@
 
     private EventAdminLogger eventAdminLogger;
 
+    private String defaultRmiServerHostname;
```

Comment:
I would rather use a name like `originalRmiServerHostname` more than `default`. This field is more to store existing `java.rmi.server.hostname` (system property)  more than a default value.